fhir-persistence-jdbc/src/test/resources/test.erase.properties

```diff
@@ -0,0 +1,22 @@
+# type is either postgres or db2
+type=postgres
+
+# Postgres
+postgres.db.host=192.168.86.29
```

Comment:
what is this IP?

Reply:
It happened to be my localhost. I can change it.

Reply:
changed